apps/models.go

```diff
@@ -83,12 +83,15 @@ type FormComponent struct {
 	// The type property will be used to select which component to render on the frontend. It cannot be an existing field type
 	Type string `yaml:"type,omitempty" json:"type,omitempty"`
 	// If true, the field will be required to have a value
-	Required  bool       `yaml:"required,omitempty" json:"required,omitempty"`
-	MinLength int64      `yaml:"minLength,omitempty" json:"minLength,omitempty"`
-	MaxLength int64      `yaml:"maxLength,omitempty" json:"maxLength,omitempty"`
-	Pattern   string     `yaml:"pattern,omitempty" json:"pattern,omitempty"`
-	TextField *TextField `yaml:"text_field,omitempty" json:"text_field,omitempty"`
-	Select    *Select    `yaml:"select,omitempty" json:"select,omitempty"`
+	Required        bool       `yaml:"required,omitempty" json:"required,omitempty"`
+	MinLength       int64      `yaml:"minLength,omitempty" json:"minLength,omitempty"`
+	MaxLength       int64      `yaml:"maxLength,omitempty" json:"maxLength,omitempty"`
+	Pattern         string     `yaml:"pattern,omitempty" json:"pattern,omitempty"`
+	TextField       *TextField `yaml:"text_field,omitempty" json:"textField,omitempty"`
+	Select          *Select    `yaml:"select,omitempty" json:"select,omitempty"`
+	ShowOnView      bool       `yaml:"show_on_view,omitempty" json:"showOnView,omitempty"`
+	ShowValueOnEdit bool       `yaml:"show_value_on_edit,omitempty" json:"showValueOnEdit,omitempty"`
+	Editable        bool       `yaml:"editable,omitempty" json:"editable,omitempty"`
```

Comment:
what pattern do we want here for snake case vs camel case? matching https://github.com/ketch-com/rest/blob/01d0eb8f85bc518ddb1de17c992b19be2e470aa7/server/models.go#L3121

Reply:
camelCase

Reply:
Should we go camel case across yaml and json? Inconsistent throughout - now would be the time to do it

Reply:
yes